File: 1304. Find N Unique Integers Sum up to Zero/code1.java
class Solution {
    public int[] sumZero(int n) {
        int arr[]=new int[n];
        int start=0;
        int last=n-1;
         for(int i=(n/2)*-1;i<((n%2==0)?0:1);i++){
            arr[last--]=-i;
            arr[start++]=i;
        }   
        return arr;
    }
}

File: 682. Baseball Game/code1.js
class Solution {
    public int calPoints(String[] ops) {
        ArrayList<Long> arr=new ArrayList<>();
        Long total=0l;
        for(String a:ops){
            Long value=0l;
            switch(a){
                case "+":
                    value=arr.get(arr.size()-2)+arr.get(arr.size()-1);
                    arr.add(value);
                    total+=value;
                    break;
                case "D":
                    value=2*arr.get(arr.size()-1);
                    arr.add(value);
                    total+=value;
                    break;
                case "C":
                    value=arr.get(arr.size()-1);
                    arr.remove(arr.size()-1);
                    total-=value;
                    break;
                default:
                    value =Long.parseLong(a);
                    arr.add(value);
                    total+=value;
                    break;
            }
        }
        return total.intValue();
    }
}